openlibrary/core/vendors.py

```diff
@@ -141,20 +202,19 @@ def _get_amazon_metadata(id_, id_type='isbn'):
     :rtype: dict or None
     """
 
-    kwargs = {}
     if id_type == 'isbn':
         id_ = normalize_isbn(id_)
-        kwargs = {'SearchIndex': 'Books', 'IdType': 'ISBN'}
-    kwargs['ItemId'] = id_
-    kwargs['MerchantId'] = 'Amazon'  # Only affects Offers Response Group, does Amazon sell this directly?
-
-    if not lending.amazon_api:
-        raise Exception("Open Library is not configured to access Amazon's API")
-    product = lending.amazon_api.lookup(**kwargs)
-    # sometimes more than one product can be returned, choose first
-    if isinstance(product, list):
-        product = product[0]
-    return _serialize_amazon_product(product)
+    try:
+        if not lending.amazon_api:
+            raise Exception
```

Comment:
Why has the info gone? This should only occur in dev envs and gives info.

Reply:
Sorry, why has what "info" gone?

Reply:
This code is called every time we fetch prices for any book